Platforms/QemuSbsaPkg/VirtNorFlashStandaloneMm/VirtNorFlashStandaloneMm.inf

```diff
@@ -0,0 +1,68 @@
+#/** @file
+#
+#  Component description file for NorFlashDxe module
+#
+#  Copyright (c) 2011 - 2021, Arm Limited. All rights reserved.<BR>

```

Comment:
This is our own inf, correct? considering add the copyright?

Reply:
Updated to include Microsoft copyright.